oasislmf/preparation/il_inputs.py

```diff
@@ -563,9 +563,22 @@ def __merge_gul_and_account(gul_inputs_df, accounts_df, fm_terms, oed_hierarchy)
     cond_class = oed_hierarchy['condclass']['ProfileElementName'].lower()
     loc_num = oed_hierarchy['locnum']['ProfileElementName'].lower()
 
-    policy_df = accounts_df.drop_duplicates(subset=[portfolio_num, acc_num, policy_num, 'layer_id']).drop(columns=cond_tag)
-    cond_df = accounts_df[[portfolio_num, acc_num, cond_tag]].drop_duplicates()
-    all_cond_policy =  pd.merge(policy_df, cond_df, on=[portfolio_num, acc_num])
+    if cond_class in accounts_df.columns:
```

Comment:
can you add a small (even one-liner) comment about the logic of what we're doing here below?
either before the `if` or inside each case, so it is clear what action has to be taken depending on whether `cond_class` is in `df.columns`.
e.g.
```py
if cond_class in accounts_df.columns:
    # in this case, we ...
else:
    # ...